actions/migrations/0001_initial.py

```diff
@@ -4,7 +4,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
I would be inclined to skip migrations in the linters, since they're mostly autogenerated by Django, and django includes this empty line. Then again, it'll be auto-fixed from now on for any new migrations, so it's not really an issue.

Reply:
Thank you for your comment. I think I'm going to format migrations, as that means `pyproject.toml` is shorter. I've seen a couple of comments about Django 5.0 formatting migrations, but couldn't find anything in the release notes.